wp-content/civi-extensions/civiglific/Civi/MonetaryReceiptService.php

```
@@ -0,0 +1,225 @@
+  public static function getSubscribedEvents() {
+    return [
+      '&hook_civicrm_post' => [
+        'triggerMonetaryEmail',
+      ],
+    ];
+  }
+
+  /**
+   *
+   */
+  public static function triggerMonetaryEmail($op, $entity, $id, $objectRef = NULL) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$op'. (undefined)

(UnusedFormalParameter)